dash/frontend/src/app/modules/private/pages/falco/falco-settings/falco-settings.component.html

```diff
@@ -90,7 +100,7 @@ <h3>Who to Notify</h3>
     </div>
 
     <div class="event-table col-sm-6">
-      <mat-card style="height:780px; min-width: 400px; display: grid;!important;">
+      <mat-card style="height:800px; min-width: 400px; display: grid;!important;">
```

Comment:
Because this feature isn't implemented, I think we should just set "display: none" to the entire mat-card.